aksel.nav.no/website/pages/templates/skjemavalidering/demo.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect } from "react";
+import { ArrowLeftIcon, ArrowRightIcon } from "@navikt/aksel-icons";
+import {
+  Button,
+  ErrorSummary,
+  HGrid,
+  Heading,
+  Page,
+  Radio,
+  RadioGroup,
+  TextField,
+  VStack,
+} from "@navikt/ds-react";
+
+const validatePersonnummer = (p: string) => {
```

Comment:
Can add a comment to this linking to [fnrvalidator](https://github.com/navikt/fnrvalidator), the recommended way to validate fnr,dnr etc in NAV